data/src/main/kotlin/com/antosik/benchproject/data/common/modules/movies/MoviesModule.kt

```diff
@@ -27,4 +29,8 @@ internal object MoviesModule {
     @Provides
     fun provideMovieDetailsRepository(apiService: MoviesApi, database: MovieDatabase): MovieDetailsRepository =
         MovieDetailsDataRepository(apiService, database.dao)
+
+    @Provides
```

Comment:
Alternatively, we could do the repo injectable and use @Binds annotation :) please check this option :D

Reply:
I used it some months ago for example here https://github.com/Nataniel-Antosik/Bench_Project/pull/10/files

```
@Module
@InstallIn(ViewModelComponent::class)
interface RepositoryModule {

    @Binds
    fun bindPopularMoviesRepository(impl: PopularMoviesDataRepository): PopularMoviesRepository
}
```

```
@Module
@InstallIn(SingletonComponent::class)
object MoviesModule {

...

    @Singleton
    @Provides
    fun providePopularMoviesDataRepository(apiService: MoviesApi) = PopularMoviesDataRepository(apiService)
}
```
And I remember the reason why I don't use `binds` is provides are shorter than creating binds and provider

Reply:
After our discussion I created two issues :D #184, #185 😄